README.md

```diff
@@ -24,131 +25,183 @@ At present time this open-source repository is maintained separately from the Ca
 * [License](#license)
 
 ## Server Setup
-Download and install the latest LTS version of node from the [Node.JS website](`https://nodejs.org/en/download/`).
+
+Download and install the latest LTS version of Node.js from the [Node.JS website](https://nodejs.org/en/download/).
```

Comment:
Feedback from customer .. confused about the specific version to download. Suggest adding v8.10.0 to this sentence.

Reply:
Updated.